v1/v1-model/src/commonMain/kotlin/com/codingfeline/twitter4kt/v1/model/user/User.kt

```diff
@@ -27,24 +27,24 @@ import kotlinx.serialization.Serializable
 public data class User(
     @SerialName("id") val id: Long,
     @SerialName("id_str") val idStr: String,
-    @SerialName("name") val name: String,
-    @SerialName("screen_name") val screenName: String,
-    @SerialName("location") val location: String?,
+    @SerialName("name") val name: String? = null,
```

Comment:
Oh, now I see what you mean.
This indeed does not looks good 🤔

Reply:
The options are to not support trimUser, do the null stuff above, or have them fill in default empty fields. I'm currently doing the nullification.